CRM/Core/OptionGroup.php

```diff
@@ -21,13 +21,23 @@ class CRM_Core_OptionGroup {
   /**
    * $_domainIDGroups array maintains the list of option groups for whom
    * domainID is to be considered.
+   *
+   * FIXME: Hardcoded list = bad. It would be better to make this a column in the civicrm_option_group table
```

Comment:
I've always wondered about declaring our option groups in files (like with settings). It would take some work to do but I think it would be easier to maintain than the current way we manage the installation & could be hookable etc